envision/web/src/components/app.js

```diff
@@ -65,9 +65,9 @@ function App({ client }) {
 
   let getSimIdsLength = async () => {
     let ids = await client.fetchSimulationIds();
-    setTotalSimIds(ids.length);
+    if (ids.length != totalSimIds) setTotalSimIds(ids.length);
   };
-  setInterval(getSimIdsLength, 3000);
+  // setInterval(getSimIdsLength, 3000);
```

Comment:
I am not quite sure why but this was causing a lot of slow-down. I think it might have to do with reflow.

Reply:
I am still uncertain what to do about this, I will likely re-enable it, but require a new issue.